### ISSUE TITLE
Audio Mixer: Most controls don't have labels

### ISSUE THREAD

**Issue by dstorey** [MicrosoftEdge/Demos-old#246]
https://developer.microsoft.com/en-us/microsoft-edge/testdrive/demos/audiomixer/

* Sliders like volume use small rather than a label
* Switches to turn on track don't have any text
* Ranges inside those tracks don't have labels (actually not sure if the label such as "kick belongs to the first switch which it is visually close to, or the slider which it precedes in the DOM (but is a DIV, not a LABEL)
* text fields such as name, author have no labels, only placeholders. Should have aria-label with same value as the placeholder if you don't want a visual label (but note in this solution, once someone starts typing the placeholder isn't visible so users might forget what the label is)
